README.md

```diff
@@ -96,3 +111,34 @@ Basic `black` commands:
 [`pre-commit`](https://pre-commit.com/) can be used to install and
 run all the formatting tools as git hooks automatically before a
 commit.
+
+
+## Running tests
+
+* Set the `DEBUG` environment variable to `1`.
+* Run `pytest`.
+
+
+## Issue tracking
+
+* [Github issue list](https://github.com/City-of-Helsinki/ahti/issues)
+* [Jira issues](https://helsinkisolutionoffice.atlassian.net/projects/AHTI/issues/?filter=allissues)
+
+
+## API documentation
+
+View the API documentation by visiting: http://localhost:8082/graphql and see
```

Comment:
we could also give a link to https://ahti-api.test.kuva.hel.ninja/graphql here? probably first step for most devs is to look at some running test instance docs if possible